config/permissions.cfg

```diff
@@ -29,9 +30,12 @@
 # Default behaviour:
 # ------------------
 #   - create missing resources if supported by the service (and tree automatically resolvable), then apply permissions.
-#   - create missing user/group if required (default user created: (group: anonymous, password: 12345).
-#   - applicable service, user or group is missing, corresponding permissions are ignored and not updated.
-#   - unknown actions are ignored and corresponding permission are not updated, unspecified action resolves to 'create'.
+#   - create missing user/group if required (default user created: (group: anonymous, password: 12345) ).
```

Comment:
Curious about this user created with hardcode password, can the user login with this hardcoded password?  Or is the user disabled until a real password is set by the admin?

Reply:
For now the user can login with that password. It is just like any other user creation. 
There is DACCS story about tmp password. This is one use case that is addressed by it.